src/Velopack/Internal/Utility.cs

```diff
@@ -163,6 +163,25 @@ public static string CalculateStreamSHA1(Stream file)
             }
         }
 
+        /// <inheritdoc cref="CalculateStreamSHA256"/>
+        public static string CalculateFileSHA256(string filePath)
+        {
+            var bufferSize = 1000000; // 1mb
+            using (var stream = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize)) {
+                return CalculateStreamSHA256(stream);
+            }
+        }
+
+        /// <summary>
+        /// Get SHA256 hash of the specified file and returns the result as a base64 encoded string (with length 44)
+        /// </summary>
+        public static string CalculateStreamSHA256(Stream file)
+        {
+            using (var sha256 = SHA256.Create()) {
+                return Convert.ToBase64String(sha256.ComputeHash(file));
```

Comment:
My only concern was around using Base64 instead of Hex for the SHA256 representation. Other tools (eg a command line utility) which take a file and compute a checksum will likely output this to hex, and I think it would be quite unusual to see a tool which produced a checksum in base64.

Reply:
God catch and fix by @SandstromErik :+1: 

I did a quick check and at least powershells get-filehash does hex encoding.
(I recently worked with for sftp fingerprint where sha256 and up does base64 encoding instead of hex which is only used for old md5)